recipe/bld.bat

```diff
@@ -4,19 +4,29 @@ if errorlevel 1 exit 1
 cd build
 if errorlevel 1 exit 1
 
+set BUILDCONF=Release
+
+set "CMAKE_COMPILER_PATH=%VSINSTALLDIR%\..\..\VC\bin\amd64"
+
 cmake -G Ninja ^
-    -D CMAKE_BUILD_TYPE=Release ^
+    -D CMAKE_CXX_COMPILER="%CMAKE_COMPILER_PATH%\cl.exe" ^
+    -D CMAKE_C_COMPILER="%CMAKE_COMPILER_PATH%\cl.exe" ^
+    -D CMAKE_LINKER="%CMAKE_COMPILER_PATH%\link.exe" ^
```

Comment:
You might need to change these to forward slashes also...

Reply:
:+1: thanks -- trying this now...

Reply:
Not sure where I was getting the `../..` from but I took it out and now they're off racing against the timeout clock :horse_racing: on Azure too